pages/migrations/0005_remove_postindexpage_tags_remove_postpage_tags_and_more.py

```
@@ -0,0 +1,25 @@
+            model_name="postindexpage",
+            name="tags",
+        ),
+        migrations.RemoveField(
+            model_name="postpage",
+            name="tags",
+        ),
+        migrations.RemoveField(
+            model_name="routablehomepage",
+            name="tags",
+        ),
+    ]
```

Ruff (0.16.0)

[warning] Mutable default value for class attribute

(RUF012)